src/components/MarketDetail/index.js

```diff
@@ -230,18 +230,6 @@ class MarketDetail extends Component {
               {moment.utc(market.eventDescription.resolutionDate).local().format(RESOLUTION_TIME.ABSOLUTE_FORMAT)}
             </div>
           </div>}
-        {showWithdrawFees && (
-          <div className="withdrawFees">
-            <div className="withdrawFees__icon icon icon--earnedTokens" />
-            <div className="withdrawFees__details">
-              <div className="withdrawFees__heading">{decimalToText(weiToEth(market.collectedFees))} {collateralTokenToText(market.event.collateralToken)}</div>
-              <div className="withdrawFees__label">Earnings through market fees</div>
-            </div>
-            <div className="withdrawFees__action">
-              <button className="btn btn-link" type="button" onClick={this.handleWithdrawFees}>Withdraw fees</button>
-            </div>
-          </div>
-        )}
```

Comment:
Did you mean to remove it completely? It's a new section now?

Reply:
Yes, now we have a new section next to 'Resolve' button, it was in the audit.